src/QueueOperation.h

```diff
@@ -142,21 +142,39 @@ class QueueDownload : public QueueOperation {
 
 class QueueDownloadHandle : public QueueOperation {
 public:
-							QueueDownloadHandle(HWND hNotify, const char * externalFile, HANDLE hFile, Transfer_Mode tMode, int notifyCode = 0, void * notifyData = NULL);
+	QueueDownloadHandle(HWND hNotify, const char* externalFile, HANDLE hFile, Transfer_Mode tMode, int notifyCode = 0, void* notifyData = NULL);

```

Comment:
In general try to avoid adding formatting stuff as part of a feature PR. So unrelated changes doesn't disturb the review of the relevant parts of the code.

Reply:
changed back